Basic/ClientDriven/Assets/Scripts/ServerPlayerMove.cs

```diff
@@ -0,0 +1,75 @@
+using System.Collections;
+using Unity.Netcode;
+using Unity.Netcode.Components;
+using Unity.Netcode.Samples;
+using UnityEngine;
+
+/// <summary>
+/// Server side script to do some movements that can only be done server side with Netcode. In charge of spawning (which happens server side in Netcode)
+/// and picking up objects
+/// </summary>
+[DefaultExecutionOrder(0)] // before client component
+public class ServerPlayerMove : ClientServerBaseNetworkBehaviour
+{
+    private ClientPlayerMove m_Client;
+
+    [SerializeField]
+    private Camera m_Camera;
+
+    public NetworkVariable<bool> ObjPickedUp = new NetworkVariable<bool>();
+
+    private void Awake()
+    {
+        m_Client = GetComponent<ClientPlayerMove>();
+    }
+
+    public override void OnNetworkSpawn()
+    {
+        base.OnNetworkSpawn();
+        if (!enabled) return;
+
+        // the following two lines should work, yet they don't. The second client connecting won't receive this position set and will spawn at the wrong position
+        // var spawnPoint = ServerPlayerSpawnPoints.Instance.ConsumeNextSpawnPoint();
```

Comment:
I think that was a bug now fixed in latest version. Client RPC were not working correctly during the NetworkSpanw not sure though.

Reply:
yeah, see conversation in sdk channel, there's still an out of order issue.

Reply:
fixed